postman/adorsys.adapter.postman_data.json

```diff
@@ -1,11 +1,11 @@
 [
   {
     "adapter": "adorsys",
-    "bankCode": "adorsys-integ",
+    "bic": "adorsys-integ",
```

Comment:
i guess BIC value should be replaced by `ADORSYS `

Reply:
yes, fixed